activerecord/lib/rails/generators/active_record/migration/templates/create_table_migration.rb.tt

```diff
@@ -6,6 +6,8 @@ class <%= migration_class_name %> < ActiveRecord::Migration[<%= ActiveRecord::Mi
       t.string :password_digest<%= attribute.inject_options %>
 <% elsif attribute.token? -%>
       t.string :<%= attribute.name %><%= attribute.inject_options %>
+<% elsif attribute.reference? -%>
+      t.references :<%= attribute.name %><%= attribute.inject_options %><%= primary_key_type ? ", type: :#{primary_key_type}" : "" %>
```

Comment:
Maybe here should be `t.<%= attribute.type %>` to take in consideration the `belongs_to`?